frameworks/PHP/laravel/benchmark_config.json

```diff
@@ -138,6 +138,29 @@
 			"display_name": "laravel-octane [frankenphp]",
 			"notes": "",
 			"versus": "php"
+		},
+		"pripple": {
+			"json_url": "/json",
+			"db_url": "/db",
+			"query_url": "/queries/",
+			"fortune_url": "/fortunes",
+			"update_url": "/updates/",
+			"plaintext_url": "/plaintext",
+			"port": 8080,
+			"approach": "Realistic",
+			"classification": "Fullstack",
+			"database": "MySQL",
+			"framework": "laravel",
+			"language": "PHP",
+			"flavor": "PHP8.3",
+			"orm": "Full",
+			"platform": "pripple",
+			"webserver": "PServer",
+			"os": "Linux",
+			"database_os": "Linux",
+			"display_name": "laravel",
```

Comment:
Please change `"display_name":` to  `"laravel [pripple]"`.
So I'll be easily identifiable in the visualize graphs, and not confunded with the plain Laravel permutation.